src/partials/footer-scripts.hbs

```diff
@@ -51,6 +51,6 @@ function loadBase() {
   addPackage('base', 'dfinity/motoko-base', '{{page.attributes.moc-base-tag}}', 'src');
 }
 </script>
-<script async src="https://download.dfinity.systems/motoko/{{page.attributes.moc-version}}/js/moc-interpreter-{{page.attributes.moc-version}}.js" onload="loadBase()">
+<script async src="{{uiRootPath}}/js/moc/moc-interpreter-{{page.attributes.moc-version}}.js" onload="loadBase()">
```

Comment:
```suggestion
<script async src="{{uiRootPath}}/js/moc/moc-interpreter-0.6.11.js" onload="loadBase()">
```
As we copy the interpreter in this repo, the external `moc-version` attribute doesn't make much sense.

Reply:
Thanks. Should we remove it from the playbook repo as well? I'm not sure if it's used anywhere else outside of this file. I don't have access to all the playbook modules. Same question about `moc-base-tag`.

Reply:
Yes, it's safe to remove `moc-version` from the playbook. We want to keep the `moc-base-tag`, it's still in use.